# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/isdn4k-utils/imake_appdef_hack.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xisdnload/Imakefile.orig	2003-01-26 17:40:58.000000000 +0100
+++ ./xisdnload/Imakefile	2003-01-26 17:44:56.000000000 +0100
@@ -1,6 +1,16 @@
 XCOMM $XConsortium: Imakefile,v 1.28 93/07/29 14:01:59 rws Exp $
         DEPLIBS = XawClientDepLibs
 LOCAL_LIBRARIES = XawClientLibs
+
+/*
+ *	This is a really ugly hack to prevent usr/X11R6/lib/X11/app-defaults
+ *	from beeing re-created and so becomes shared between this package and
+ *	xfree86. Maybe it would be better to patch the imake template files to
+ *	not recreate existing symbolic links like this one.   - Clifford
+ */
+#undef LinkConfDirectory
+#define LinkConfDirectory(a,b,c,d)
+
 #if defined(SunArchitecture) && defined(i386Architecture)
   SYS_LIBRARIES = -lkvm
 #endif
